### ISSUE TITLE
Test various Mobile and Browsers

### ISSUE THREAD

**Issue by tuan91** [Cropster/ember-visual-test#29]
Hi, is it possible to test various mobile devices  and browser?
If so, how can I do that?
Thank you!

**Comment by mydea** [comment]
Not really, I fear. This uses Headless Chrome to capture the screenshots, so it is always Chrome which actually renders the page.
